tests/in/bounds-check-image-rzsw.wgsl

```diff
@@ -0,0 +1,55 @@
+[[group(0), binding(0)]]
```

Comment:
A bit unfortunate that this source file is exactly the same. Maybe we need some way of having multiple parameter files separately

Reply:
That sounds like a good idea. I'll file it as a follow-up issue, and handle it in a separate PR, if that's all right.

Reply:
Filed as #1284.

Reply:
(feel free to 'unresolve' if you'd like this handled in this PR)